chap-cheri-riscv.tex

```diff
@@ -472,21 +472,26 @@ \subsubsection{Capability Control and Status Registers (CCSRs)}
 \begin{figure}[!h]
 \begin{center}
 \begin{bytefield}[bitwidth=\textwidth/34]{32}
-  \bitheader[endianness=big]{0,1,2,31} \\
-  \bitbox{32}{\textbf{WIRI}}
+  \bitheader[endianness=big]{0,30,31} \\
+  \bitbox{1}{\texttt{tc}}
+  \bitbox{1}{\rotatebox{90}{\texttt ndr}}
```

Comment:
"ndr" doesn't easily fit in a single-bit box, so I rotated the label 90 degrees.

Reply:
That sounds reasonable. The only other thing I could think of is ~D~, but that looks weird.

Should we reserve the "No PCC relocation bit" here as well?

Reply:
Oh, I didn't know you envisioned a separate bit for PCC.  If you intend a separate bit, then we should add it, yes.  Alternatively, we could use a single 'NR' ("no relocation") bit (which would also fit in the box) to mean that both DDC and PCC relocation is disabled.  I don't think we expect the feature bit here to really be independent for DDC and PCC?

Reply:
I'd be happy to have a single bit - I just used two bits for the two separate PRs.

Reply:
Ok, I've renamed the bit to `nr` in this version.

Reply:
Done in https://github.com/CTSRD-CHERI/sail-cheri-riscv/pull/77